creator-node/src/ImageProcessingQueue.ts

```diff
@@ -1,19 +1,19 @@
-const { Queue, QueueEvents, Worker } = require('bullmq')
-const path = require('path')
-const os = require('os')
+import type { PrometheusRegistry } from './services/prometheusMonitoring/prometheusRegistry'
```

Comment:
same tiny nit as the other PR with the space here - either way for consistency but again not important ofc